src/base/ctrl_socket.c

```diff
@@ -487,15 +494,17 @@ void xdebug_control_socket_setup(void)
 		NULL
 	);
 
+	xdfree(name);
+
 	if (XG_BASE(control_socket_h) == INVALID_HANDLE_VALUE) {
 		errno = WSAGetLastError();
-		xdebug_log_ex(XLOG_CHAN_CONFIG, XLOG_WARN, "CTRL-SOCKET", "Can't create control Named Pipe (%x)", errno);
+		xdebug_log_ex(XLOG_CHAN_CONFIG, XLOG_WARN, "CTRL-SOCKET", "Can't create control Named Pipe (0x%x)", errno);
 		xdfree(XG_BASE(control_socket_path));
 		XG_BASE(control_socket_path) = NULL;
 		return;
 	}
 
-	xdebug_log_ex(XLOG_CHAN_CONFIG, XLOG_INFO, "CTRL-OK", "Control socket set up successfully: '%s'", XG_BASE(control_socket_path));
+	xdebug_log_ex(XLOG_CHAN_CONFIG, XLOG_INFO, "CTRL-OK", "Control socket set up successfully: '\\\\.\\pipe\\%s'", XG_BASE(control_socket_path));
```

Comment:
If you `xdfree(name)` after this, you don't have to re-introduce the `\\\\.\\pipe\\` part, and you can just use `name` instead. But you also will have to free it before the `return;` in line 495/504.